medusa/tv/episode.py

```diff
@@ -974,6 +975,15 @@ def delete_episode(self):
 
         raise EpisodeDeletedException()
 
+    def get_manual_search_results(self, manual_search_type='episode', show_all_results=0, perform_search=0):
```

Comment:
that code returns all that. but by design manual search was designed to be episode search or season search. As both does different stuff as the cached result from episode is different from season result